docs/content/deployment/guides/aws.mdx

```diff
@@ -26,25 +30,37 @@ storage:
       port: 5432
 ```
 
-In this case, you'll want to ensure you provide the right connection strings for your RDS instance, and ensure that the node or container hosting Dagit is able to connect to RDS.
+In this case, you'll want to ensure that:
+
+- You provide the right connection strings for your RDS instance
+- The node or container hosting Dagit is able to connect to RDS
+
+Be sure that this file is present and `DAGSTER_HOME` is set on the node where Dagit is running.
 
-Be sure that this file is present, and _DAGSTER_HOME_ is set, on the node where Dagit is running.
+**Note**: Using RDS for run and event log storage doesn't require that Dagit be running in the cloud. If you're' connecting a local Dagit instance to a remote RDS storage, verify that your local node is able to connect to RDS.
 
-Note that using RDS for run and event log storage does not require that Dagit be running in the cloud. If you are connecting a local Dagit instance to a remote RDS storage, double check that your local node is able to connect to RDS.
+---
 
 ## Deploying in ECS
 
 <CodeReferenceLink filePath="examples/deploy_ecs" />
 
-The Deploying on ECS example on GitHub demonstrates how to configure the [Docker Compose CLI integration with ECS](https://docs.docker.com/cloud/ecs-integration/) to manage all of the required AWS resources that Dagster needs to run on ECS. The example includes a Dagit container for loading and launching jobs, a `dagster-daemon` container for managing a run queue and submitting runs from schedules and sensors, a Postgres container for persistent storage, and a container with user job code. The Dagster instance uses the <PyObject module="dagster_aws.ecs" object="EcsRunLauncher" /> to launch each run in its own ECS task.
+The Deploying on ECS example on GitHub demonstrates how to configure the [Docker Compose CLI integration with ECS](https://docs.docker.com/cloud/ecs-integration/) to manage all of the required AWS resources that Dagster needs to run on ECS. The example includes:
+
+- A Dagit container for loading and launching jobs
+- A `dagster-daemon` container for managing a run queue and submitting runs from schedules and sensors
+- A Postgres container for persistent storage
+- A container with user job code. The [Dagster instance](/deployment/dagster-instance) uses the <PyObject module="dagster_aws.ecs" object="EcsRunLauncher" /> to launch each run in its own ECS task.
```

Comment:
The sentence about the run launcher isn't directly related to the sentence before it - probably should live outside of the bullet list